### DISCUSSION TITLE
Error: "fee.amount[0].amount" must be a string

### DISCUSSION THREAD

**Discussion by zhengwei-gif** [ping-pub/explorer#548] [Q&A]
what i can do?

**Comment by liangping** [comment]
"20000000"

**Reply by safeina86** [reply]
@liangping does not work
![Screenshot 2024-02-27 133254](https://github.com/ping-pub/explorer/assets/80200634/d2e939fe-23e2-43d4-98e3-3031365139c8)

**Comment by liangping** [comment]
Please reconnect your wallet
